bin/lang-js/src/function_kinds/management.ts

```diff
@@ -17,19 +18,19 @@ export interface ManagementFunc extends Func {
   thisComponent: ComponentWithGeometry;
   components: {
     [key: string]: ComponentWithGeometry;
-  }
+  };
 }
 
 export type ManagementFuncResult =
   | ManagementFuncResultSuccess
   | ManagementFuncResultFailure;
 
 export interface ManagmentConnect {
-  from: string,
+  from: string;
   to: {
     component: string;
     socket: string;
-  }
+  };
 }
 
 export interface ManagementCreate {
```

Comment:
looks like this is no longer used and can be deleted?

Reply:
ah, a bad rebase on my part I think